### ISSUE TITLE
parallelization over chromosomes

### ISSUE THREAD

**Issue by subwaystation** [nf-core/pangenome#37]
# nf-core/pangenome feature request

Hi there!

## Describe the solution you'd like
I want to be able to start the pipeline with a folder of FASTAs as an input. All current steps should be run on each of the FASTAs. This helps 

- Add new input parameter `--input-folder`
- Make sure that `--input` and `--input folder` can't be set at the same time
- Ensure that the output in the results folder reflects the naming of the input FASTA file
- Add tests

**Comment by subwaystation** [comment]
We won't parallelize over Chromosomes, but over disconnected components. Which usually are chromosomes.
Closing this. The disconnected components will be tracked by another issue.

**Comment by mictadlo** [comment]
Any updates on it?

**Comment by subwaystation** [comment]
There are 2 ways parallelize:

1. You run the pipeline in community detection mode with `--communitites`. The idea is that similar and related sequences are clustered into the same community and for each community the graph construction can be run in parallel.
2. You split your sequences manually into chromosomal communities by a given reference and execute nf-core/pangenome for each reference community.
